jll/L/librdkafka_jll/Compat.toml

```diff
@@ -1,12 +1,8 @@
-[1]
+[1-2]
 OpenSSL_jll = "1.1.10-1"
 
 ["1-1.5"]
 julia = "1"
 
 ["1.8-2"]
-JLLWrappers = "1.2.0-1"
 julia = "1.6.0-1"
```

Comment:
Why this is gone?

Reply:
Well, JLLWrappers, commenting on the phone is hard

Reply:
Oops, I put it back